src/game.h

```diff
@@ -9,33 +9,40 @@ class Game
     Game();
     ~Game();
 
-    void init(std::string build_path);
+    // This class is not ought to be copied or assigned!
```

Comment:
A game should not be copyable. Hence disable the assignment / move and copy ctors!